project_one.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="project_one.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <title>Project One</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="card">
+        <section class="snap">
+          <img src="./images/Snap.png" alt="img" />
+        </section>
+        <section class="main">
+          <h1>Tonic</h1>
+          <div class="mid">
+            <span class="can"> CANOPY </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> BACK END DEV </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> 2015 </span>
+          </div>
+          <p class="para">
```

Comment:
- [ ] Please use a `section ` tag as a container of  the work section so your code be semantically true, 
generally, section tag is intended to be used as a sectioning tag while if you want to use an element as a wrapper you can use the `div` tag